.pipelines/cni/cilium/cilium-cni-load-test.yaml

```diff
@@ -0,0 +1,147 @@
+              sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+              echo "installing helm"
+              curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+              chmod 700 get_helm.sh
+              ./get_helm.sh
+            name: "setup"
+            displayName: "Setup Kubectl and helm"
+          - task: AzureCLI@1
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                set -ex
+                az extension add --name aks-preview
+                az group create --location $(LOCATION) --name $(RESOURCE_GROUP) --subscription $(SUBSCRIPTION_ID)
+                az account set -s $(SUBSCRIPTION_ID)
+                # Create a vnet
+                az network vnet create -g $(RESOURCE_GROUP) -n overlayvnet --address-prefixes 10.0.0.0/8 --output none
+                subnet=$(az network vnet subnet create -g $(RESOURCE_GROUP) --vnet-name overlayvnet --name nodenet --address-prefix 10.10.0.0/16 -o none --query '{id: id}' --output tsv)
+                # create AKS Cluster
+                az aks create -n $(RESOURCE_GROUP) -g $(RESOURCE_GROUP) -l $(LOCATION) --node-count 10 --network-plugin azure --network-plugin-mode overlay --pod-cidr 192.168.0.0/16 --vnet-subnet-id $subnet -s Standard_D8_v3 --enable-cilium-dataplane --no-ssh-key --yes
+                az aks get-credentials -n $(RESOURCE_GROUP) -g $(RESOURCE_GROUP)
+            name: "CreateAksCluster"
+            displayName: "Create AKS Cluster"
+  - stage: pod_deployment
+    dependsOn: creating_aks_cluster
+    displayName: "Pod Deployment"
+    jobs:
+      - job: deploy_pods
+        steps:
+          - task: AzureCLI@1
+            displayName: "Pod Deployment"
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                set -ex
+                az extension add --name aks-preview
+                az aks get-credentials --resource-group $(RESOURCE_GROUP) --name $(RESOURCE_GROUP)
+                bash cni/scripts/scale_deployment.sh
+  - stage: validate_state
+    dependsOn: pod_deployment
+    displayName: "Validate State"
+    jobs:
+      - job: validate_state
+        steps:
+          - script: |
+              set -ex        
+              echo "installing kubectl"
+              curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+              sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+              sudo apt-get install jq

```

Comment:
same question

Reply:
still wondering ^ is jq not on the runner already?

Reply:
Yeah, it is already on the runner. Removed it.